src/utils/remark-rehype-presenter-notes.js

```diff
@@ -0,0 +1,27 @@
+import zone from 'mdast-zone';
+import unistVisit from 'unist-util-visit';
+import * as mdast from 'mdast-builder';
+
+export default function remarkRehypePresenterNotes(noteCallback) {
+  return tree => {
```

Comment:
Can we maybe move the functional code to the bottom of this function?

It was hard for me at least to understand it on first glance, to figure out exactly what those functions below are, that they're hoisted, and then that they're used above.

Reply:
Sounds good!